src/server/game/Spells/SpellInfo.h

```diff
@@ -181,6 +181,11 @@ enum SpellCustomAttributes
     SPELL_ATTR0_CU_REQ_CASTER_BEHIND_TARGET      = 0x00020000,
     SPELL_ATTR0_CU_ALLOW_INFLIGHT_TARGET         = 0x00040000,
     SPELL_ATTR0_CU_NONE6                         = 0x00080000,   // UNUSED
+    /*
+     * A binary spell is a spell that can't be partially resisted. Frostbolt is a binary spell,
+     * it will either hit with full damage or get fully resisted for 0 damage.
+     * Fireball is a non-binary spell and it can be half-resisted causing reduced damage.
+    */
```

Comment:
nice! it's good to have more documentation in the code

Reply:
Imagine having to figure that out by yourself!

Reply:
There's also google.
And according to https://wowwiki.fandom.com/wiki/Binary_spell?direction=prev&oldid=2475744 Frostbolt is not a binary 
Rev 11 Jan 2009